paymentsheet-example/src/androidTestDebug/java/com/stripe/android/TestGooglePay.kt

```diff
@@ -86,7 +86,7 @@ class TestGooglePay {
             testParameters.copy(
                 googlePayState = GooglePayState.On
             ),
-            R.string.stripe_paymentsheet_or_pay_using
+            R.string.stripe_paymentsheet_or_pay_with_card
```

Comment:
These don’t run on BrowserStack, but I noticed that they’re broken when running on my physical device.